client/reader/discover/components/add-new/index.tsx

```diff
@@ -14,6 +15,15 @@ import './style.scss';
 const DiscoverAddNew = () => {
 	const translate = useTranslate();
 	const isEmailVerified = useSelector( isCurrentUserEmailVerified );
+	const [ hasFeedPreview, setHasFeedPreview ] = useState< boolean >( false );
+
+	const onChangeFeedPreview = useCallback( ( hasPreview: boolean ): void => {
+		setHasFeedPreview( hasPreview );
+	}, [] );
+
+	const onSubscribeToggle = useCallback( (): void => {
+		setHasFeedPreview( false ); // Close the feed preview when the subscription is toggled.
+	}, [] );
```

Comment:
I notice in the corresponding reddit tab we also dispatch requestFollows here, should we do that here on add-new as well?

Reply:
We are requesting all subscriptions while rending Subscriptions table so doing same here make double requests which are unnecessary.

Reply:
Currently same issue exist on `/reader/subscriptions` where adding a feed doesn't update the "Recent" dropdown. We can fix it separately.

I will add this issue in our Bugs board.